workflows/src/main/kotlin/com/r3/corda/lib/ci/InitiatingFlows.kt

```diff
@@ -0,0 +1,108 @@
+package com.r3.corda.lib.ci
+
+import co.paralleluniverse.fibers.Suspendable
+import com.r3.corda.lib.tokens.workflows.internal.flows.confidential.RequestConfidentialIdentityFlow
+import com.r3.corda.lib.tokens.workflows.internal.flows.confidential.RequestConfidentialIdentityFlowHandler
+import net.corda.core.flows.*
+import net.corda.core.identity.AbstractParty
+import net.corda.core.identity.Party
+import net.corda.core.identity.PartyAndCertificate
+import net.corda.core.transactions.WireTransaction
+import java.security.PublicKey
+import java.util.*
+
+/**
+ * Initiating version of [RequestConfidentialIdentityFlow].
+ */
+@InitiatingFlow
+@StartableByRPC
+class ConfidentialIdentityInitiator(private val party: Party) : FlowLogic<PartyAndCertificate>() {
```

Comment:
File looks good but not sure about the names of the flows... Why can't this just be `RequestConfidentialIdentity` and the other `RequestConfidentialIdentityHandler` ?

Reply:
Done